experiments/opencog/pattern_matcher_vqa/pattern_matcher_vqa.py

```diff
@@ -181,52 +171,102 @@ def __str__(self):
 class PatternMatcherVqaPipeline:
 
     def __init__(self, featureExtractor, questionConverter, atomspace, answerHandler):
-        self.logger = logging.getLogger('PatternMatcherVqaPipeline')
         self.featureExtractor = featureExtractor
         self.questionConverter = questionConverter
         self.atomspace = atomspace
         self.answerHandler = answerHandler
+        self.formulaQuestionMap = dict()
+        self.logger = logging.getLogger('PatternMatcherVqaPipeline')
+        tmp_map = self.questionConverter.getFormulaQuestionMap()
+        for item in tmp_map.entrySet():
+            self.formulaQuestionMap[item.getKey()] = item.getValue()
```

Comment:
I would propose add method to the QuestionToOpencogConverter to return all necesary information:
- formula
- questionType
- schema query